documentation/source/library-reference/system/file-system.rst

```diff
@@ -407,6 +408,49 @@ File-System module.
 
      - :func:`delete-directory`
 
+.. generic-function:: create-symbolic-link
+
+   Creates a symbolic link
+
+   :signature: create-symbolic-link *target* *link* => ()
+
+   :param target: An instance of `<pathname>`:class:.
+   :param link:   An instance of `<pathname>`:class:.
```

Comment:
Does this syntax work?  We usually write ```:class:`<pathname>` ```

Reply:
Yes, hmmm. The Yoda way of documentation, it is. Correct it, I will.